sys/transceiver/transceiver.c

```diff
@@ -672,7 +672,7 @@ void receive_nativenet_packet(radio_packet_t *trans_p)
 void receive_at86rf231_packet(ieee802154_packet_t *trans_p)
 {
     DEBUG("Handling AT86RF231 packet\n");
-    dINT();
+    unsigned int mask = disableIRQ();
```

Comment:
for consistency, how about changing these to `unsigned state` or adopting the pattern for existing places (at least `receive_nativenet_packet`) ?